pkg/virt-api/webhooks/arm64.go

```diff
@@ -27,12 +27,6 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 )
 
-var _false bool = false
-
-const (
-	defaultCPUModelArm64 = v1.CPUModeHostPassthrough
```

Comment:
I see one usage of it in `setDefaultArm64CPUModel`.

Reply:
I was mistakenly looking at `pkg/defaults/arm64.go` that looks very similar.
Sorry for the noise.